lib/win32/certstore/certificate/certificate.rb

```diff
@@ -0,0 +1,48 @@
+#
+# Author:: Piyush Awasthi (<piyush.awasthi@msystechnologies.com>)
+# Copyright:: Copyright (c) 2017 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'cert_base'
+require 'store/store_base'
+
+module Win32
+  class Certstore
+    class Certificate
+      include Certificate::CertBase
+      include Win32::Store::StoreBase
+
+      attr_accessor :certstore_name
+
+      def initialize(certstore_name)
+        @certstore_name = certstore_name.first
+      end
+
+      # To Display Certificate List
+      # Take input certificate store name
+      # Return List in JSON format
+      def list

```

Comment:
see note above. I'd expect to use the certificate store to list certs and not a Certificate class. Since this is the only method here, I wonder if we need this class at all.

Reply:
@mwrock , yeah `list` may be moved to CertStore since it doesn't represent a single certificate instance. But there will be other methods for adding, deleting, verifying certificates, etc which will make sense in this class. Please correct me if I missing something.

Reply:
I still think methods like adding and deleting certs from a store belong in the store and would take a cert instance.